### PR TITLE
API-1302 application details page for approved applications

### PR DESCRIPTION
**Pull request by edgarjimenez**

### CODE REVIEW

File: app/controllers/DashboardController.scala [hunk 1]
```diff
@@ -92,9 +92,34 @@ trait DashboardController extends FrontendController with GatekeeperAuthWrapper
...
+
+      def application(app: ApplicationResponse, approved: StateHistory, admins: Seq[User], submissionDetails: SubmissionDetails) = {
+        val verified = app.state.name == State.PRODUCTION
+        val details = applicationDetails(app, submissionDetails)
+
+        ApprovedApplication(details, admins, approved.actor.id, approved.changedAt, verified)
+      }
+
+      for {
+        app <- applicationConnector.fetchApplication(appId)
```
Comment (tamaslang) [inline app/controllers/DashboardController.scala]: I recommend extract it to a private method like it was done for the review app page.

File: app/views/approvedApplication/approved.scala.html [hunk 2]
```diff
@@ -1,55 +1,80 @@
...
-        <summary class="status--verified">Verified</summary>
-        <div>
-            <p class="panel-indent">The submitter has verified that they are still associated with the email address associated with this application.</p>
+
+    @defining(DateTimeFormat.forPattern("dd MMMM yyyy")) { dateFormatter =>
+        <div class="content__body">
+
+            <nav class="breadcrumb-nav">
+                <ul>
+                    <li class="breadcrumb-nav__item"><a href="dashboard">Application names</a></li>
```
Comment (tamaslang) [inline app/views/approvedApplication/approved.scala.html]: Isn't the link text dashboard?

Comment (tamaslang) [inline app/views/approvedApplication/approved.scala.html]: +href should point to dashboard too

File: app/controllers/DashboardController.scala (hunk 1 above)
Comment (edgarjimenez) [inline app/controllers/DashboardController.scala]: This code requires the application with history, which is not returned by the new fetchApplicationDetails method. Also, this is not used anywhere else, so can't see any benefit of extracting it.

Comment (tamaslang) [inline app/controllers/DashboardController.scala]: Still there are common characteristic in application initialisation which could be reused as opposed to copypaste. 
But as discussed it is not the priority now.

File: app/views/approvedApplication/approved.scala.html (hunk 2 above)
Comment (edgarjimenez) [inline app/views/approvedApplication/approved.scala.html]: The l link actually points to dashboard. The only problem is the text used, which is also used in the review page. I've raised the question and we'll update it once we know what the value should be for both pages.

Comment (edgarjimenez) [inline app/views/approvedApplication/approved.scala.html]: The requirement is that the text should actually be 'Application names'. Leaving it as is.

**Pull request merged**